model/db.js

```diff
@@ -239,60 +239,60 @@ async function deinitDb() {
 }
 
 async function populateDb() {
-  await user.createUserAppAccount('one@a.com', '98765432', 'one', 'one', db)
-  await user.createUserAppAccount('two@a.com', '88765432', 'two', 'two', db)
-  await user.createUserAppAccount('thr@a.com', '78765432', 'thr', 'thr', db)
-  await user.createUserAppAccount('fou@a.com', '68765432', 'fou', 'fou', db)
-  await user.createUserAppAccount('fiv@a.com', '58765432', 'fiv', 'fiv', db)
-  await user.createUserAppAccount('alpha@a.com', '98213212', 'six', 'six', db)
-  await user.createUserAppAccount('bravo@a.com', '95513212', 'sev', 'sev', db)
-  await user.createUserAppAccount('charlie@a.com', '94765432', 'eig', 'eig', db)
-  await user.createUserAppAccount('delta@a.com', '93765432', 'nin', 'nin', db)
-  await user.createUserAppAccount('echo@a.com', '92765432', 'ten', 'ten', db)
-  await user.createUserAppAccount('foxtrot@a.com', '88565432', 'ele', 'ele', db)
-  await user.createUserAppAccount('golf@a.com', '81165432', 'twe', 'twe', db)
-  await user.createUserAppAccount('hotel@a.com', '81155432', 'thir', 'thir', db)
+  await user.createUserAppAccount('one@a.com', '11765432', 'one', 'one_pw', db)
```

Comment:
A number that starts with 8 or 9 will be preferred lest this is deliberately trying to test the constraint

Reply:
Deliberate